docs/README.md

```diff
@@ -21,6 +21,7 @@
   * [Gardenlet](concepts/gardenlet.md)
 * [Backup Restore](concepts/backup-restore.md)
 * [etcd](concepts/etcd.md)
+* [Kubernetes API Server](concepts/kubernetes-apiserver.md)
```

Comment:
Maybe this should rather be moved to the `operations` or `usage`, or maybe even `development` folder? This way, the introduced inconsistency in the filenames for Gardener components can also be avoided.

Reply:
Initially, I added it as a concept to be equivalent for the etcd autoscaling docs: https://github.com/gardener/gardener/blob/v1.94.0/docs/concepts/etcd.md#autoscaling

But I agree that this is rather development docs. Hence, I moved it under development and reverted the other renaming changes.